Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/AWSIdentityPropertyKeys.swift

```diff
@@ -26,4 +26,14 @@ public enum AWSIdentityPropertyKeys {
     /// Used only in conjunction with the `awsv4-s3express` auth scheme, which generates bucket-specific credentials
     /// for use with the S3 Express service.
     public static let s3ExpressClient = AttributeKey<any S3ExpressCreateSessionClient>(name: "AWSIdentityS3Client")
+
+    /// The list of credential feature IDs saved to a credentials object's properties.
+    /// Used internally by credential resolvers to store credential feature IDs during credential resolution.
+    public static let credentialFeatureIDs = AttributeKey<[String]>(name: "CredentialFeatureIDs")
+
+    /// The list of credential feature IDs saved to operation context of internal service clients by `CredentialFeatureIDInterceptor`.
+    /// Used to include feature IDs resolved so far in profile chain in service calls made by relevant credential resolvers.
+    public static let manuallyConfiguredCredentialFeatureIDs = AttributeKey<[String]>(
+        name: "ManuallyConfiguredCredentialFeatureIDs"
+    )
```

Comment:
This key is used to save manual configuration for credential feature ID into operation context of `InternalAWSXXX` clients when service-client dependent credential resolvers make service requests. It's used to inject list of credential feature IDs resolved so far into the user agent.